cmd/pimo/main.go

```diff
@@ -63,13 +63,20 @@ var (
 	repeatWhile           string
 	statisticsDestination string
 	statsTemplate         string
+	statsDestinationEnv   = os.Getenv("PIMO_STATS_URL")
```

Comment:
maybe change to PIMO_STATS_OUTPUT (URL seems a bit confusing to me)

Reply:
LINO use LINO_STATS_URL too